CHANGELOG.md

```diff
@@ -110,6 +110,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   are now better at adding Git refs to prevent that.
   [#815](https://github.com/martinvonz/jj/issues/815)
 
+* The warning printed when no user information has been provided now ends with
+  a terminating newline.
+
```

Comment:
I think this change is too small to be worth mentioning. It's not something people will have run into repeatedly, and it's unlikely that people have been waiting for this fix before they try jj. I'm still fine with keeping it if you prefer.

Reply:
Yup, sounds good. I wasn't sure whether I should add it or not but did it after I got prompted by the PR template. Let's remove it; this is a (single byte) change :)